WChat/src/app/components/RootNavigator/components/headers.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Image, Text, Platform } from 'react-native';
+import { HeaderBackButton } from 'react-navigation';
+import styles from './styles';
+
+export const IosHeader = ({goBack, title, avatar}) => (
```

Comment:
Instead of creating two headers, one for each OS, define a single component, and inside of it, decide what to show depending on the Platform

Reply:
you changed it in the other pr, so nevermind 👍